### ISSUE TITLE
PIN - MOD UNDER MAINTANANCE

### ISSUE THREAD

**Issue by shedaniel** [shedaniel/BetterLoadingScreen#49]
Currently I am taking an exam and am not able to mod! I will be back in a month or so.

This mod at the current state is **NOT** good for any modpacks or modding setup! I have rewritten 80% of the new backend that will be more compatible and up to date with new standards!

### **Please hold off for bug reports and version port requests!**

**Comment by tt-thoma** [comment]
Yeah just take your time, don't feel obligated to answer to any issue or to fix some problems of your mod's problems

**Comment by ashley-graves** [comment]
I don't mean to rush but how is the exam going

**Comment by ItsProfessional** [comment]
> I will be back in a month or so.

It's been 5 months. I don't mean to rush but I'm too impatient for this mod lol

**Comment by dmgnr** [comment]
Same
